ddtrace/tracer/telemetry.go

```diff
@@ -0,0 +1,76 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package tracer
+
+import (
+	"fmt"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/telemetry"
+)
+
+// start the global instrumentation telemetry client with tracer data
+func startTelemetry(c *config) {
+	// need to re-intialize default values
```

Comment:
Just noticed calling `applyDefaultOps` and checking if the telemetry client has started is redundant since `GlobalClient.Start` already does these things, so I will remove them and add more logging inside `Start`

I originally had these checks when I tried supporting both the tracer and profiler starting the telemetry client, but now they are unnecessary.